board/common/rootfs/usr/share/udhcpc/default.script

```diff
@@ -105,7 +105,13 @@ case "$ACTION" in
 	    /usr/sbin/avahi-autoipd -c $interface && /usr/sbin/avahi-autoipd -k $interface
 	fi
 
-	if /bin/ip addr add dev $interface $ip/$NETMASK $BROADCAST proto 5; then
+	if [ -n "$NETMASK" ]; then
+	    IP=$ip/$NETMASK
+	else
+	    IP=$ip
```

Comment:
```suggestion
	    IP=$ip/32
```
I had to try this manually (address without prefix) to see what iproute2 would do. Suggest we make it clear to future readers what will happen in this case.

Reply:
Good point, committed your suggestion, but will probably set the default NETMASK earlier in a follow-up.